src/bp-templates/bp-nouveau/js/customizer.js

```diff
@@ -93,6 +93,44 @@ window.wp = window.wp || {};
 		$( '#sub-accordion-panel-bp_mailtpl' ).on( 'click', '.customize-panel-back', function() {
 			location.replace( BP_Customizer.platformCustomizerUrl );
 		} );
+
+
+		$( 'ul#customize-control-profile-header' ).sortable( {
+			cursor    : 'move',
+			axis      : 'y',
+			opacity   : 1,
+			items     : 'li:not(.ui-sortable-disabled)',
+			tolerance : 'intersect',
+
+			update: function() {
+
+				var order = [];
+
+				$( this ).find( '[data-bp-nav]' ).each( function( s, slug ) {
+					order.push( $( slug ).data( 'bp-nav' ) );
+				} );
+
+				if ( order.length ) {
+					$( '#bp_profile_header_order' ).val( order.join() ).trigger( 'change' );
+				}
+
+			}
+		} ).disableSelection();
+
+		$( document ).on( 'change', '#bp_profile_header_order', function() {
+			var buttons = $( this ).val();
+
+
+			$.post( ajaxurl, { buttons : buttons, action : 'save_profile_header_buttons_order' }, function(){
+
+			} );
+
+		});
```

Comment:
@shohelbuddyboss the controls in customizer are listen by the WP customizer js lib. can you check if we don't do this without checking with actual wp customizer. There should be a hidden or named field to store values for the order and it will sync automatically with customizer.

Reply:
@mehulkaklotar 

This is now deprecated as the changes are now syncing with customizer automatically.